worker.js

```diff
@@ -231,15 +231,74 @@ function Worker() {
             console.log(error);
           });
       };
-      this[`${quiz}.allForumTopic`] = () => {
+      this[`${quiz}.allForumPost`] = () => {
         return db
-          .model('ForumTopic')
+          .model('ForumPost')
           .fetchAll()
           .then(data => {
             return data.toJSON();
           });
       };
-      this[`${quiz}.getAllStimuli`] = () => {
+      this[`${quiz}.createForumPost`] = data => {
+        return new Model(data)
+          .save()
+          .then(resp => {
+            return resp.toJSON();
+          })
+          .catch(error => {
+            console.log(error);
+          });
+      };
+      // this[`${quiz}.updateQuizUserWithAuth0`] = (uid, auth0uid) => {
+      //   if(auth0uid){
+      //     return db.model('User').where({id: uid}).set({auth0_id: auth0uid})
+      //   }
+      // }
+      this[`${quiz}.generateUser`] = auth0_id => {
+        if (auth0_id) {
+          return db
+            .model('User')
+            .where({ auth0_id: auth0_id })
+            .fetch()
+            .then(data => {
+              if (!data) {
+                return db
+                  .model('User')
+                  .forge()
+                  .save({
+                    created_at: new Date(),
+                    updated_at: new Date(),
+                    auth0_id: auth0_id
+                  })
+                  .then(user => {
+                    return user;
+                  });
```

Comment:
`.then(data => { return data })` is unnecessary.

Reply:
check